### ISSUE TITLE
Khoa's Tasks (2024-10-14)

### ISSUE THREAD

**Issue by ArsenalAlex108** [NamSellsFish/circle-help#1]
- [ ] Standardize and document Authentication and Authorization API and below APIs.
- [x] **Standardize and document get stock by row API**
~~- [ ] Research architecture and frameworks (Micronaut) **(Report by Wednesday)**~~
- [x] Deploy this service and other service ~~by microservice way~~
- [x] Update code related to `GetCompartmentProducts` API to generate new response body (see in Postman, you don't need to implement any algorithm to get those statuses, just simply add a status field (data type is number, and we will implicitly understand its meaning) to the table and some mock data. However, after manually or automatically sorting which changes the positions of the stocks, you need to implement a piece of code to check and update the status accordingly)
- [x] Implement manual stock arrangement API (inventory -> shelves, swap shelves, shelves -> inventory)
- [ ] Implement automatic stock arrangement API (basic, holidays/events)
![ui](https://github.com/user-attachments/assets/24a75817-a75b-48fb-8934-700d7eabd2e1)
[references.for.tasks.of.week.14-10.md](https://github.com/user-attachments/files/17444574/references.for.tasks.of.week.14-10.md)
- [x] Change ID => SKU
- [x] Add compartmentNoFromUserPerspective to API
- [ ] Prepare algorithm description and demonstration script
- [x] Convert manual stock arrangement to a single endpoint

**Comment by NamSellsFish** [comment]
### You have not done any of above bullets, please now continue to work with them, PLUS:
- [ ] Create apis for use case "View stock on shelf and inventory", similar to [this](https://www.realworld.how/specifications/backend/endpoints/#list-articles) (prior shelf first, inventory is nice to have for this week). 
- [ ] Document and test them using Postman
- [ ] Deploy this service and other service by microservice way
**Prerequisite:** finish previous tasks,
